src/scss/components/popout.scss

```diff
@@ -123,12 +135,43 @@
             }
         }
     }
+}
+
+
+@media (min-width: $tablet) {
+    // The main container.
+    body > .login-section, .container {
+        min-width: 361px;
+        max-width: 400px;
+    }
 
     .widget.contacts {
-        .empty-list .contact,
-        .not-found-contacts .contact {
-            height: 48px;
-            box-sizing: border-box;
+        width: 400px;
+
+        .contact {
+            width: 400px;
+        }
+    }
+}
+
+@media (max-width: $tablet) {
```

Comment:
If you remove the `display: flex;` from the .list on smaller viewports, it'll just scroll in height instead.